src/java/org/streamingpool/core/service/impl/LocalPool.java

```diff
@@ -50,7 +54,9 @@
 public class LocalPool implements DiscoveryService, ProvidingService, StreamFactoryRegistry {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(LocalPool.class);
+    private static final int DEFAULT_THREAD_POOL_SIZE = 100;
```

Comment:
This I find confusing ... why is there a second default here? There is already one in the configuration, injected by a property....